tests/test_embedding_offline.py

```
@@ -0,0 +1,52 @@
+
+def test_run_mcp_server_sets_hf_hub_offline_before_server_import():
+    script_path = Path(__file__).parent.parent / "run-mcp-server.py"
+    script_code = script_path.read_text()
+
+    fake_globals = {
+        "__name__": "__main__",
+        "__file__": str(script_path),
+    }
+    real_import = builtins.__import__
+
+    def fake_import(name, globals=None, locals=None, fromlist=(), level=0):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `fake_import`

(ANN202)